net.retrodeck.retrodeck.yml

```diff
@@ -86,7 +86,7 @@ modules:
     sources:
       - type: git
         url: https://github.com/XargonWan/RetroDECK.git
-        branch: cooker
+        branch: main
```

Comment:
This should use a specific commit, not a branch.

Reply:
I understand but I opted to do the main one as I don't need to update it every time.
However the main branch is PRed only after the cooker testing, so it's the stable and "safe" one.

Techinally: when te builder starts it doesn't know what is the latest commit's hash, so a generic "main" will catch that, otherwise it would be always one cmmit behind and the builder will always fail as it will work on old assets.

Reply:
This logic is backwards. Manifest is supposed to produce reproducible builds, and you can't have that if you point it to an arbitrary git branch which can change at any time. This is something that will start to be eventually rejected by buildbot.